deploy/clowdapp.yaml

```
@@ -64,6 +64,17 @@ objects:
           requests:
             cpu: ${AVAILABILITY_LISTENER_CPU_REQUEST}
             memory: ${AVAILABILITY_LISTENER_MEMORY_REQUEST}
+      readinessProbe:
+        httpGet:
+          path: /health
+          port: 8000
+        initialDelaySeconds: 1
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 8000
+          initialDelaySeconds: 10
+        periodSeconds: 30
```

```suggestion
        readinessProbe:
          httpGet:
            path: /health
            port: 8000
          initialDelaySeconds: 1
        livenessProbe:
          httpGet:
            path: /health
            port: 8000
          initialDelaySeconds: 10
          periodSeconds: 30
```

sorry, this is super complex. this _should_ work. this way you can click apply and it should take the diff for you.